configure.ac

```diff
@@ -219,7 +219,8 @@ if test x"$debug_enabled" != x"yes" ; then
     CFLAGS="-DNDEBUG $CFLAGS"
 fi
 
-LDFLAGS="-Wl,-z,defs $LDFLAGS"
+dnl LDFLAGS="-Wl,-z,defs $LDFLAGS"
```

Comment:
Not sure I want to have that commented out line in the code.

Also what is this about? It removes -z,defs from the ldflags.
This means allow undefined symbols in object files. That
sounds like papering over real issues...

What is the reason for the removal?
If it is really needed, I'd like to have it conditional.
(only remove it for osX).